### ISSUE TITLE
Discord update solution

### ISSUE THREAD

**Issue by literallyfabin** [mrtolkien/inhouse_bot#42]
discord had an update in which the bot stopped working, which in this case commands like prefix ''!, %, $'' don't work. Would you be able to update to commands like /queue top ?

**Comment by spiralwind** [comment]
Same issue for me as well. As all commands on discord with the bot don't work anymore

**Comment by yenoww** [comment]
let me know if you find a solution please

**Comment by spiralwind** [comment]
So the fix to this in to add " INHOUSE_BOT_COMMAND_PREFIX: COMMAND_HERE(default is !)" in your docker_compose.yml file and run docker-compose up so it recreates it and your all good

**Comment by mrtolkien** [comment]
As written in the `README.md`:

> As `discord.py` is not maintained anymore and the LCS player association has developed its own matchmaking system, this bot is not maintained anymore as of December 2021.

It was known that Discord was developping breaking changes that would make the bot require a rewrite unfortunately. I don't have the time to handle that at the moment.

I'll leave the thread open so people can see it here too, but a good title would help.

**Comment by literallyfabin** [comment]
To end this discussion, the solution that worked was as follows:
putting "intent.messages = True" in inhouse_bot.py file

(Along with what our friend spiralwind said above)

So the fix to this in to add "INHOUSE_BOT_COMMAND_PREFIX: COMMAND_HERE(default is !)" in your docker_compose.yml file and run docker-compose up so it recreates it and your all good

With that the bot works again through the command ''/''

**Comment by yenoww** [comment]
but wouldn't I have to have cloud access to modify the inhouse_bot.py file?
If it's possible to fix the bot and you can help me.. call me on discord Rodrigoo#1939 (I'm Brazilian)

**Comment by adryel97** [comment]
Conseguiram resolver o bot do inhouse bot?
Did you manage to solve the inhouse bot bot?

**Comment by adryel97** [comment]
Coloquei INHOUSE_BOT_COMMAND_PREFIX: COMMAND_HERE(default is !) e não voltou a funcionar.

I put INHOUSE_BOT_COMMAND_PREFIX: COMMAND_HERE(default is !) and it didn't work again.

**Comment by DBVSF** [comment]
@literallyfabin conseguiu resolver? não sei como modificar a .py ja que é no docker. Tem como dar uma força ? possuo um servidor grande e preciso colocar o bot do inhouse pra funcionar. Me chama no disc CBLIXO#0068 te dou uma grana se ajudar!

**Comment by Euphener** [comment]
yeah, what are the details needed to login to adminer?

**Comment by spiralwind** [comment]
> yeah, what are the details needed to login to adminer?

it was failed to mention but It the same process as done on the youtube  tutorial but he forgot to add for the 
 that the username: postgres

**Comment by Euphener** [comment]
> 

Yeah sorry, I did that and it still doesn't work :/ , it just says "connection refused"

**Comment by spiralwind** [comment]
you need to make sure that the you wrote everything right so below
System: PostgreSQL
Server: db
username: postgres
password : whatever you set in the POSTGRES_PASSWORD:____________ in docker-compose
 database: whatever you set in  POSTGRES_DB:___________ in docker-compose

**Comment by Euphener** [comment]
> you need to make sure that the you wrote everything right so below System: PostgreSQL Server: db username: postgres password : whatever you set in the POSTGRES_PASSWORD:____________ in docker-compose database: whatever you set in POSTGRES_DB:___________ in docker-compose

You reckon you could add me on disc? Sorry, but I only have one more slight issue zee#6902

**Comment by Euphener** [comment]
> you need to make sure that the you wrote everything right so below System: PostgreSQL Server: db username: postgres password : whatever you set in the POSTGRES_PASSWORD:____________ in docker-compose database: whatever you set in POSTGRES_DB:___________ in docker-compose

thank you man, helped out a lot

**Comment by Luccaop** [comment]
someone has able to find how to edit the inhouse_bot.py? to put "intent.messages = True" in inhouse_bot.py file

> To end this discussion, the solution that worked was as follows: putting "intent.messages = True" in inhouse_bot.py file
> 
> (Along with what our friend spiralwind said above)
> 
> So the fix to this in to add "INHOUSE_BOT_COMMAND_PREFIX: COMMAND_HERE(default is !)" in your docker_compose.yml file and run docker-compose up so it recreates it and your all good
> 
> With that the bot works again through the command ''/''

**Comment by NoatakLoL** [comment]
Do you know any alternatives or a fix? Trying to set this up

>

**Comment by Cxacho** [comment]
> To end this discussion, the solution that worked was as follows: putting "intent.messages = True" in inhouse_bot.py file
> 
> (Along with what our friend spiralwind said above)
> 
> So the fix to this in to add "INHOUSE_BOT_COMMAND_PREFIX: COMMAND_HERE(default is !)" in your docker_compose.yml file and run docker-compose up so it recreates it and your all good
> 
> With that the bot works again through the command ''/''

Could you please provide more info on how to edit the file, since we do not have the permission to edit files in the original repo?

**Comment by Tr3vvv** [comment]
In the discord development portal, in the guide you have to enable 'Server Member Intent', you must also enable 'Message Content Intent' this is the important step!

In docker compose, under the emojis make sure you enter this in the same tab line,
INHOUSE_BOT_COMMAND_PREFIX: '!'  
MAKE SURE YOUR COMMAND IS IN SINGLE TICKS

You can change the command to whatever you want, by default it is '!'.
